content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,196 @@
+---
+title: DVC 3 years anniversary and 1.0 pre-release
+date: 2020-05-04
+description: |
+  Today, we’ve got three big things to announce: 🎉 3rd-year anniversary of DVC,
+  🚀 DVC 1.0 pre-release is ready and ⭐ 5000 GitHub starts.
+
+descriptionLong: |
+  Today, we’ve got three big things to announce.
+    - 🎉 3rd-year anniversary of DVC
+    - 🚀 DVC 1.0 pre-release is ready
+    - ⭐ DVC has reached 5K GitHub starts (coincidently on the same day)
+
+  We are sharing our learnings from this journey and how they affected the new
+  DVC 1.0 release.
+picture: /static/uploads/images/2020-05-04/5k_stars.png
```

Comment:
just 2020-05-04/5k_stars.png should do the trick

where did you get that full path btw? should we update some docs?

Reply:
done